vmimport_policy.tf

```diff
@@ -0,0 +1,140 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policies that give the vmimport service role sufficient
+# permissions to function in the Images (Production) and Images (Staging)
+# accounts.
+# ------------------------------------------------------------------------------
+
+# These policy documents are based on the permissions required for the vmimport
+# service role as described in
+# https://docs.aws.amazon.com/vm-import/latest/userguide/vmie_prereqs.html#vmimport-role
+data "aws_iam_policy_document" "vmimport_production" {
+  # Buckets to use for image import (VM -> AMI)
+  statement {
+    actions = [
+      "s3:GetBucketLocation",
+      "s3:GetObject",
+      "s3:ListBucket",
+    ]
+    resources = [
+      data.terraform_remote_state.assessment_images.outputs.assessment_images_bucket_production.arn,
+      "${data.terraform_remote_state.assessment_images.outputs.assessment_images_bucket_production.arn}/*"
+    ]
+  }
+
+  # Buckets to use for image export (AMI -> VM)
+  statement {
+    actions = [
+      "s3:GetBucketLocation",
+      "s3:GetObject",
+      "s3:ListBucket",
+      "s3:PutObject",
+      "s3:GetBucketAcl",
```

Comment:
Please sort these actions alphabetically.  The same goes for a few other similar blocks below and in other policy files.